src/mtbl_merge.c

```diff
@@ -329,6 +333,16 @@ parse_arg_compression_level(const char *arg)
 	return *endp == '\0';
 }
 
+static bool
+parse_arg_thread_count(const char *arg)
+{
+	size_t thread_count = atoi(arg);
```

Comment:
use int not size_t.
use strtol() instead of atoi, modeled on the function above this.
fail if thread_count <= 0

Reply:
Should thread_count fail if <= 0?  Creating a threadpool with 0 threads is valid, because all it will do is explicitly disable multithreading.

Reply:
ok, just fail if < 0.